app/lib/dttp/params/contact.rb

```diff
@@ -46,10 +41,23 @@ def build_params
           "gendercode" => GENDER_CODES[trainee.gender.to_sym],
           "dfe_EthnicityId@odata.bind" => "/dfe_ethnicities(#{contact_dttp_ethnicity_id})",
           "dfe_DisibilityId@odata.bind" => "/dfe_disabilities(#{contact_dttp_disability_id})",
-          "parentcustomerid_account@odata.bind" => "/accounts(#{trainee.provider.dttp_id})",
         }
       end
 
+      def append_create_only_params
+        create_only_params = {
+          "dfe_ContactTypeId@odata.bind" => "/dfe_contacttypes(faba11c7-07d9-e711-80e1-005056ac45bb)", # trainee type
```

Comment:
Feels like this ID should be in a constant.

Reply:
Oh aye... forgot we'd discussed that. I've updated.